### PR TITLE
Remove unused state

### PR DESCRIPTION
**Pull request by gruberb**
We are not using our local state. I can see why we would it leave in there (in case we need it at some point), but let's remove unused code.

### CODE REVIEW

File: src/DialpadPlugin.js [hunk 1]
```diff
@@ -23,14 +21,4 @@ export default class DialpadPlugin extends FlexPlugin {
 
     this.registerReducers(manager);
```
Comment (trogers-twilio) [inline src/DialpadPlugin.js:22]: Need to remove this line of code. Plugin is currently throwing an error here and failing to load.

```
twilio-flex.unbundled-react.min.js:1476 6/29/2021, 2:16:37 PM - PluginManager: There was an error during 'flex-dialpad-addon-plugin' plugin initialization

Original error:
"this.registerReducers is not a function"
```

**Review by trogers-twilio — CHANGES_REQUESTED** [review]
The code cleanup is much appreciated. Once the suggested change is made, I'll re-test and merge if all looks good.

File: src/DialpadPlugin.js (hunk 1 above)
Comment (gruberb) [reply src/DialpadPlugin.js]: Done!

**Review by trogers-twilio — APPROVED** [review]

**Comment by trogers-twilio** [comment]
Successfully tested changes.

**Pull request merged**